### DISCUSSION TITLE
More than one instance of Jamulus at the same time

### DISCUSSION THREAD

**Discussion by jhsa** [jamulussoftware/jamulus#1575] [Ideas]
I am using Jamulus to rehearsal with my band during this Pandemic. it has been a life saver by the way, so thank you very much for this wonderful project.
We normally record our rehearsal and our drummer starts a Metronome most of the times. The problem is, sometimes the recordings come out quite good but the Metronome is also recorded together with his electronic drumset, and we cannot remove it.
I have found a workaround which is to start another instance of Jamulus, and select another different virtual ASIO driver from "Voicemeeter Potato" as the device on this second instance of Jamulus. Like this We can have a metronome showing up as a new user, and it will be recorded separately on it's own track. Like this we can also set the metronome volume to a much pleasant level, and not blow our ears :)
Now, for this second instance I set a different virtual ASIO driver and a different user name. The problem is the jamulus only saves the last instance to be closed. So, if I close the metronome instance last, next time i start Jamulus, it will start with these settings, and the settings from my first instance will be lost. This means that every time we want to rehearsal I have to redo all the settings. If it would be possible to save different instances (Profiles?) it would be really nice and make our life even easier.. I tried  "File/Save Mixer Channels Setup", but these parameters aren't saved..
Thank you in advance for your answer, and thank you so much for this amazing tool :)

João

**Comment by chrisrimple** [comment]
You can create two Shortcuts in Windows to Jamulus, each specifying a different INI file so that settings from each instance are stored separately.  I'm on Mac so can't provide specifics, but the instructions have been published many times and are likely in the Wiki.

**Reply by jhsa** [reply]
Thanks Chris, it is great to know that it might be already possible to do it, Will check it out and will come back here afterwards.

**Reply by jhsa** [reply]
Hmm, can't find anything about this :(

**Comment by jhsa** [comment]
Found a little tutorial showing how to do it here:

https://sourceforge.net/p/llcon/discussion/533517/thread/2ac46dc38a/?limit=100

Thank you so much.

Also, this might help others. On my windows 7 computer the Jamulus.ini file is in:
C:\Users\MyUserName\AppData\Roaming\Jamulus\Jamulus.ini

I copied the "Jamulus.ini" file and called it "Jamulus2.ini". then placed it in the same folder.

Then followed the instructions on the link I posted above.
